packages/cli/src/lib/vite-config.ts

```diff
@@ -31,7 +32,19 @@ export async function isViteProject(root: string) {
   return isVite;
 }
 
-export async function getViteConfig(root: string, ssrEntryFlag?: string) {
+type ViteConfigResult = {
+  clientOutDir: string;
+  serverOutDir: string;
+  serverOutFile: string;
+  resolvedViteConfig: any;
+  userViteConfig: any;
```

Comment:
Isn't it possible to get these types from Vite? `import type {...} from 'vite'`. That should not break bundling or anything if it's just types 🤔